src/app/shared/components/event-rate-limit/style.scss

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'variables';
+
+.no-margin {
+  margin: 0 !important;
```

Comment:
Do not use `!important` modifier. We are trying to get rid of it at all from the codebase as it often causes conflicts and makes it hard to debug.